test/integration/machine-learning.spec.ts

```diff
@@ -101,6 +105,58 @@ describe('admin.machineLearning', () => {
         });
     });
 
+    it('creates a new Model with valid AutoML TFLite ModelFormat', function () {
+      const modelOptions: admin.machineLearning.ModelOptions = {
+        displayName: 'node-integ-test-create-automl',
+        tags: ['tagAutoml'],
+        tfliteModel: { automlModel: 'this will be replaced below' }
```

Comment:
Why not initialize this whole thing below?

Reply:
Because it's being used in both the createModel and then one function level down in the verifyModel. I wouldn't have access to it in verifyModel if it was in the createModel scope I don't think? (Unless I'm wrong?)

Reply:
```
return getAutoMLModelReference()
  .then((automlRef) => {
    if (!automlRef) {
      this.skip();
    }

    const modelOptions = {}; // init the whole thing here
    // create and verify
  })
```

Reply:
Apparently I'm wrong. Done.